docs/content/guides/styling/themes/react/exampleTheme.jsx

```diff
@@ -1211,56 +1213,105 @@ export const data = [
 ];
 
 const ExampleComponent = () => {
+  const hotRef = useRef(null);
+  const [themeName, setThemeName] = useState(
+    document.querySelector('html')?.classList.contains('theme-dark')
+      ? 'ht-theme-horizon-dark'
+      : 'ht-theme-horizon'
+  );
+
+  const handleOnChange = (event) => {
+    setThemeName(event.target.value);
+  };
+
+  useEffect(() => {
+    if (!hotRef.current?.hotInstance) return;
```

Comment:
I'd prefer always using brackets here
```suggestion
    if (!hotRef.current?.hotInstance) {
      return;
    }
```